xenit-alfresco/templates/acs/acs-service.yaml

```diff
@@ -20,10 +20,11 @@ spec:
         protocol: TCP
         port: 8443
         targetPort: 8443
-    {{- if ((((.Values.acs).email).inbound).port) }}
+    {{- if and ((((.Values.acs).email).inbound).servicePort) ((((.Values.acs).email).inbound).port) }}
```

Comment:
This should do the trick, but why so many brackets?

There are values named "inbound.serviceport" and not "inbound"."serviceport"?

Lastly (but this will make the chart less readable)
We could add a **required** function like "setting the inbound mailport requires you to set the serviceport?"
Might also have to do it the other way around and make the whole thing overly complex.

Reply:
Without the braces, rendering fails if any of the nested levels are nil -which is the case if we leave the value empty by default. https://stackoverflow.com/a/68807258

I held back on that to keep things simple, putting conditional requirements seems like a recipe for disaster in the templates. It's also a niche function and Ive been verbose on the explanation, so I'd count that as enough

Reply:
Alright, did not know this, good to know.